pkg/operator/ceph/nfs/config.go

```diff
@@ -24,7 +24,9 @@ const (
 	userID         = "admin"
 )
 
-func getGaneshaConfig(spec cephv1beta1.NFSGaneshaSpec, nodeID string) string {
+func getGaneshaConfig(n cephv1beta1.NFSGanesha, name string) string {
+	spec := n.Spec
+	nodeID := n.Name + "." + name
```

Comment:
i'd recommend the following as the more common for golang, and maybe even have a helper method for it since you form the id in several places
```
nodeID := fmt.Sprintf("%s.%s", n.Name, name)
```